### ISSUE TITLE
Skin 

### ISSUE THREAD

**Issue by djdirty60** [Tikipeter/repository.tikipeter#484]
Which skin works best with this addon?

Or will there be another skin of your own?  Will be top notch....

**Comment by Tikipeter** [comment]
I could skin the whole addon, but that would be a tonne of work and then no users would be able to set their own look the way they like by installing skins. Doing little things like the new Extras window are about as far as I should go really. Mixing python with skinning is a lot of work too though, so it took a long while.

As for skins, most of the main ones should be OK. Whilst I really like the Arctic skins from jurial, when I've used them I've turned off ALL external calls the skin's helpers make for extra info. That slows the browsing experience down a lot, and people would likely often think it's the addon being slow, when in fact it's a skin helper making calls for info. Also the more views a skin offers for each media type, the slower it will be. That's the main reason Kodi's default skin (Estuary) is very fast. There's only a couple of viewtype options for each media type.

Apart from the speed issue, really it's a personal choice. The only thing to be on the lookout for when trying to match skin with addon is the fonts the skin provides. Addons like Fen (and Venom and Seren) that make some custom window xml's for display can use their own graphic files, but are reliant on the fonts provided by the skin currently being used. So if I code a label to use `font12`, and `font12` isn't included in the skin currently being used, then the label will not show correctly. That is a problem when matching skins with addons if the addon uses some custom xml windows.

In the end, though, it always comes down to personal preference.
